.github/workflows/CI-linux.yml

```
@@ -121,4 +121,9 @@ jobs:
           set -x
           sudo bash -c 'echo core.%p.%p > /proc/sys/kernel/core_pattern'
           ulimit -c unlimited
-          ${{ matrix.config.qemu }} make -j8 check
+          CROSS_LIB="/usr/${{ matrix.config.host }}"
+          make -j8 check LOG_DRIVER_FLAGS="--qemu-arch ${{ matrix.config.qemu }}" LDFLAGS="-L$CROSS_LIB/lib -static" QEMU_LD_PREFIX="$CROSS_LIB"
```

Thanks! It looks good to have more CI coverage now.

In very odd coincidental timing, I was also just thinking back today about how to fix this, so I found today that you could do here the same as this with the existing `libtool` script:
```suggestion
          LOG_COMPILER='../libtool --mode=execute qemu-${{ matrix.config.target }}'
          TESTS_ENVIRONMENT='LD_LIBRARY_PATH=/usr/${{ matrix.config.target }}-linux-gnu/lib'
          make -j8 check
```

This it might also need to explicitly install the expected shared libraries earlier (in the install QEMU step)
```
          sudo apt install qemu-user ${{ matrix.config.toolchain }} ${{ matrix.config.libc }} -y
          sudo ln -s /usr/${{ matrix.config.host }}/lib/ld-linux-${{ matrix.config.target }}.so.1 /lib/
```
given the new variable matrix.config.libc

---

The libc and dynamic loader are installed as dependencies of the qemu-user package so there's no need to explicitly install them. 

Yes, it's possible to use LOG_COMPILER+libtool instead, for tests that are simple binary executables, but we already had a hook for replacing LOG_DRIVER which is required for running tests on remote targets (like for embedded OSes) and I was familiar with it. Six of one half a dozen of the other but a customer LOG_DRIVER script will be a more elegant solution to issue #403 in the future.